### ISSUE TITLE
feat(server_info): update gateway status

### ISSUE THREAD

**Issue by zaunist** [apache/apisix-dashboard#2348]
# Feature request

## Please describe your feature

The Apache APISIX `server-info` plugin alreay update, we need update manager-api too.
Ref https://github.com/apache/apisix/pull/6202

## Describe the solution you'd like

We need remove this two filed in the dashboard form

<img width="1613" alt="image" src="https://user-images.githubusercontent.com/38528079/155663519-8b7737f3-3262-4b33-b6a8-3837904e75f3.png">

## Additional context

The current `server-info` plugin uses keepalive to replace the `put` operation , so these two fields are not needed and we need to remove them from the form. 

I have taken a preliminary look at the backend and I don't think it needs to be modified, but I haven't verified it so far, so I'll verify later if the back end also needs to be modified in parallel.

**Comment by bzp2010** [comment]
Since the Admin API does not provide such an API, this feature will not be implemented in the next renovation. ref #2981 

If you have any questions, feel free to start a new issue!
